miniapp/miniapp_reduction_to_band.cpp

```diff
@@ -153,6 +183,7 @@ int main(int argc, char** argv) {
   desc_commandline.add_options()
     ("matrix-size",  value<SizeType>()   ->default_value(4),      "Matrix rows")
     ("block-size",   value<SizeType>()   ->default_value(2),      "Block cyclic distribution size")
```

Comment:
Please use some more realistic default params...

Reply:
I would go for our classic `matrix-size=4096` and `block-size=256`.

About `band-size`? Do you like having it defaulted to `block-size`? or would you prefer to have a smaller number just to have a more "reasonable" use-case?

Reply:
we can leave it as nb for now and change later when we have a good choice for the eigensolver.
Anyway, does the distributed case handle nb != b, otherwise I would add a fallback in the options check.